changelogs/Spyder-6.md

```diff
@@ -360,12 +360,30 @@ In this release 25 pull requests were closed.
 
 ----
 
-## Version 6.0.8 (Unreleased)
+## Version 6.0.8 (2025/08/26)
+
+### Important fixes
+
+* Fix input method positioning for the Editor and IPython Console.
 
 ### API changes
 
 * Add `spyder_conf_version` and `old_spyder_conf_version` properties to `SpyderConfigurationAccessor`.
 
+### Issues Closed
+
+* [Issue 24561](https://github.com/spyder-ide/spyder/issues/24561) - Input method is not located in the right position in Spyder 6 ([PR 24575](https://github.com/spyder-ide/spyder/pull/24575) by [@athompson673](https://github.com/athompson673))
```

Comment:
Please remove this issue and the entry for PR 24575 from the 6.1.0a4 changelog because they are now part of 6.0.8.